eng/pipelines/templates/stages/vmr-build.yml

```diff
@@ -858,6 +858,24 @@ stages:
             - Windows_x86
             - Windows_arm64
 
+        - template: ../jobs/vmr-build.yml
+          parameters:
+            buildName: Windows
+            isBuiltFromVmr: ${{ parameters.isBuiltFromVmr }}
+            vmrBranch: ${{ variables.VmrBranch }}
+            pool: ${{ parameters.pool_Windows }}
+            targetOS: windows
+            targetArchitecture: x86
+            runTests: false
```

Comment:
Remove this line as we decided to enable running tests by default for all verticals. Even if those are just unit tests for tasks.

```suggestion
```